### PR TITLE
Update rmblast to version 2.10.0

### PR DESCRIPTION
**Pull request by nathanweeks**
The latest version of RMBlast is 2.10.0: http://www.repeatmasker.org/RMBlast.html

### CODE REVIEW

File: recipes/rmblast/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/{{ version }}/ncbi-blast-{{ version }}+-src.tar.gz
-  sha256: a390cc2d7a09422759fc178db84de9def822cbe485916bbb2ec0d215dacdc257
+  sha256: 3acdd9cec01c4f43e56aeaf89049cb8f8013d60b9c1705eced10166967f1d926
   #patches:
```
Comment (bgruening) [inline recipes/rmblast/meta.yaml:11]: patches are not used anymore, isn't it? So we can remove them?

Comment (nathanweeks) [reply recipes/rmblast/meta.yaml:11]: I thought the "Needs the -p1 option, applied in build.sh" comment was helpful in providing the rationale for why the patch is applied in build.sh, but it's fine to remove the (commented-out) patches field entirely from meta.yaml if that's preferred.

**Comment by Juke34** [comment]
@bgruening I tried without the patch and it does not work. see here: https://github.com/bioconda/bioconda-recipes/commit/e5b8fa936d5de6a2e1b9a0db614c31d006668e34

So we can let as it is here and merge

**Comment by nathanweeks** [comment]
@BiocondaBot please add label

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
Oh, the last build was too long ago for there to still be artifacts. Not a problem

**Pull request merged**